### ISSUE TITLE
Players swapping boards will be stuck if DM does not follow

### ISSUE THREAD

**Issue by Trisscar05** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#1006]
**Describe the bug**
If a player attempts to swap to a board before the DM does so, they will be stuck on a loading "Waiting for DM" screen. This in itself would not be an issue, however the player now cannot swap back, and in fact cannot do anything at all besides quit out of the campaign and rejoin.
Would like to recommend that the board list be accessible while "waiting", unless that is what is supposed to happen and therefore it's a bigger issue than I thought.

**To Reproduce**
1. Make a campaign with at least two boards, one DM, and at least one player (possibly more...?)
2. Have the player swap to a board that is not the "landing board" without the DM following.
3. Attempt to have the player do literally anything at all besides leave.

**Additional context**
On Ubuntu 21.04, through Steam's Proton.

**Comment by LuPro** [comment]
That the player can't switch back is definitely a bug, however devs said they plan to overhaul the board selection system so players can't switch on their own at all and get moved around by the GM which would fix this issue - plans on this are not final though so possibly there'll be changes on this front

**Comment by baggers-br** [comment]
@Trisscar05 yeah we definitely have some ugly bugs around the 'waiting for DM' screen. I'm off for a week but when I get back I should take a look at this. It will be removed eventually (as @LuPro indicated), but the work to get there is fairly extensive.

Thanks for the report!
